app/code/Magento/Catalog/Test/Mftf/ActionGroup/AddToCartFromCmsPageUsingAjaxActionGroup.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AddToCartFromCmsPageUsingAjaxActionGroup">
```

Comment:
Please, refer to the [best practices](https://github.com/magento/magento-functional-tests-migration/wiki/Best-practices) we've discussed previously. There's a mention regarding naming. The correct name will be

`StorefrontAddToCartFromCmsPageUsingAjaxActionGroup`

Reply:
Done it. Thanks